client/src/components/SprintPlanning/TeamCapacityTable/TeamCapacityTable.tsx

```diff
@@ -0,0 +1,190 @@
+          <TableCell>
+            <b>Position</b>
+          </TableCell>
+          <TableCell align="left">
+            <b>email</b>
+          </TableCell>
+          <TableCell align="left">
+            <b>absences</b>
+          </TableCell>
+          <TableCell align="left">
+            <b>workdays</b>
+          </TableCell>
+          <TableCell align="left">
+            <b>coefficient</b>
+          </TableCell>
+          <TableCell align="left">
+            <b>capacity</b>
+          </TableCell>
+        </TableRow>
+      </TableHead>
+    );
+  }
+  renderTableBody(): Array<JSX.Element> {
+    return this.sortTeamMembers().map((member) => {
+      const { absences, workdays, coefficient, capacity } = this.getDetails(member);
+      return (
+        <TableRow key={member.id} sx={{ "&:last-child td, &:last-child th": { border: 0 } }}>
+          <TableCell component="th" scope="row">
+            {member.position ? member.position.position : PositionsEnum.noPosition}
+          </TableCell>
+          <TableCell align="left">{member.email}</TableCell>
+          <TableCell align="left">{absences}</TableCell>
+          <TableCell align="left">{workdays}</TableCell>
+          <TableCell align="left">{coefficient}</TableCell>
+          <TableCell align="left">{capacity}</TableCell>
+        </TableRow>
+      );
+    });
+  }
+
+  renderTableFooter(): JSX.Element {
+    const totalCapacity = this.calculateTotalCapacity();
+
+    return (
+      <TableRow>
+        <TableCell align="left"></TableCell>
+        <TableCell align="left"></TableCell>
+        <TableCell align="left"></TableCell>
+        <TableCell align="left"></TableCell>
+        <TableCell align="left">
+          <b>Total</b>
+        </TableCell>
+        <TableCell align="left">
+          <b>{totalCapacity}</b>
+        </TableCell>
+      </TableRow>
+    );
+  }
+
+  getDetails(teamMember: IUserWithTeamAndPosition): TeamCapacityTableDetails {
+    const absences = this.props.absenceDays.get(teamMember.id);
+
+    if (absences === undefined) {
+      throw new Error(`User with id ${teamMember.id} does not exist`);
+    }
+
+    const workdays = this.props.sprintTotalWorkdays - absences;
+    const coefficient = teamMember.position ? teamMember.position.coefficient : 0;
+    const capacity = +(workdays * coefficient).toFixed(2);
+
+    return { absences, workdays, coefficient, capacity };
+  }
+
+  calculateTotalCapacity(): number {
+    return this.props.teamMembers.reduce((totalCapacity, member) => {
+      const { capacity } = this.getDetails(member);
+
+      return (totalCapacity += capacity);
+    }, 0);
+  }
+
+  sortTeamMembers(): Array<IUserWithTeamAndPosition> {
+    return this.props.teamMembers
+      .map((teamMember) =>
+        teamMember.position
+          ? teamMember
+          : { ...teamMember, position: { id: "", position: PositionsEnum.noPosition, coefficient: 0, sortOrder: 15 } },
+      )
+      .sort((a, b) => {
```

Comment:
Comparing can be done like this

```suggestion
      (a, b) => b.position.sortOrder - a.position.sortOrder || a.email.toUpperCase() < b.email.toUpperCase()
```

Reply:
done